packages/spark-core/tools/_mixins.scss

```diff
@@ -18,3 +18,43 @@
     margin-left: auto;
   }
 }
+
```

Comment:
I know this is nit-picky, but can you add a `//` here for consistency?

Reply:
just added